container/image.bzl

```diff
@@ -342,6 +298,7 @@ _attrs = dict({
     ),
     "label_file_strings": attr.string_list(),
     "empty_files": attr.string_list(),
+    "container_layers": attr.label_list(providers = [LayerInfo]),
```

Comment:
Emm... this is tricky since `layers` is also used in `lang_image` rule, which extends the attrs from `container_image` rule. 

So if we want to keep `layers` in `container_image` rule, we need to change the `layers` in `lang_image` to something else. 

Do you have some suggestions for me?

Reply:
I don't think this is a problem.  `lang_image` wraps `dep_layer` and `app_layer` rules which extend the `container_image` rule.  I don't believe `layers` is passed through.